nat-lab/tests/test_proxy.py

```diff
@@ -31,7 +31,7 @@ async def test_proxy_endpoint_map_update() -> None:
         )
 
         async with Ping(alpha_connection, beta.ip_addresses[0]).run() as ping:
-            await testing.wait_long(ping.wait_for_next_ping())
+            await testing.wait_lengthy(ping.wait_for_next_ping())
```

Comment:
Just curious, why such big of a difference from main branch?
`this branch`: 19 seconds
`main branch`: 11 seconds

Reply:
Good question! I can get it back to ~10s with this little trick:
```
diff --git a/crates/telio-utils/src/exponential_backoff.rs b/crates/telio-utils/src/exponential_backoff.rs
index 275d80b..9e3aa6a 100644
--- a/crates/telio-utils/src/exponential_backoff.rs
+++ b/crates/telio-utils/src/exponential_backoff.rs
@@ -88,7 +88,7 @@ impl Backoff for ExponentialBackoff {
     }

     fn next_backoff(&mut self) {
-        self.current_backoff *= EXPONENTIAL_BACKOFF_MULTIPLIER;
+        self.current_backoff = self.current_backoff + (self.current_backoff / 3);

         if let Some(maximal) = self.bounds.maximal {
             if self.current_backoff > maximal {
```
And after some digging it seem to me that this PR introduces some small delay (expected) but it aligns in an unfortunate way some of the other timers in the `proxy.rs` that now jump up to the 4s range thanks to the exponential growth. Imo, this is not a real problem but let me know if you think I should do something here.

Reply:
It's all good, you have a valid explanation. I tought some bug might have been introduced